lib/stack_master/validator.rb

```diff
@@ -13,6 +13,7 @@ def perform
     rescue Aws::CloudFormation::Errors::ValidationError => e
       $stderr.puts "Validation Failed"
       $stderr.puts e.message
+      false
```

Comment:
A lot of commands don't care about their return value. I'd prefer it if we added this to command and called `failed` here, then collected `command.perform(..).success?` in CLI to determine if all commands succeeded.

```
    module ClassMethods
      def perform(*args)
        new(*args).tap { |command| command.perform }
      end
    end

    def failed
      @failed = true
    end

    def success?
      @failed != true
    end
```